### PR TITLE
[ECO-1372] Add research from Uni v3 whitepaper

### PR DESCRIPTION
**Pull request by alnoki**
# PR history

This originally used as a base branch #45 , then #45 merged. Note that commits up to 0e77b28 are replicated in that PR, and this PR only adds content starting at later commits.

So as not to re-write history, after #45 merged, `main` was merged into here

# Changes

1. Add detailed overview of Uniswap v3 whitepaper key equations
2. Add sample structs adapated to commentary on v3 whitepaper
3. Update package name for a previous research installment
4. Update dictionary file for global terms

### CODE REVIEW

File: src/move/research/uniswap-v3-whitepaper/sources/structs.move [hunk 1]
```diff
@@ -0,0 +1,53 @@
+module research::structs {
+
+    use aptos_framework::table::Table;
+
+    struct Ratio has copy, drop {
+        numerator: u64,
+        denominator: u64,
+    }
+
+    /// An NFT.
```
Comment (xbtmatt) [inline src/move/research/uniswap-v3-whitepaper/sources/structs.move:10]: Is this struct only for an NFT? The description in the readme doesn't make it seem like it, but not sure

**Review by xbtmatt — APPROVED** [review]
Just the question about the NFT comment, other than that lgtm!

File: src/move/research/uniswap-v3-whitepaper/sources/structs.move (hunk 1 above)
Comment (alnoki) [reply src/move/research/uniswap-v3-whitepaper/sources/structs.move:10]: The `Position` struct is an NFT that represents a user's liquidity contribution. In Uni v2 these were fungible tokens, but in v3 they are not fungible. But you can still borrow against them, sell them, etc., hence NFT

Comment (xbtmatt) [reply src/move/research/uniswap-v3-whitepaper/sources/structs.move:10]: Ahhh okay gotcha

**Pull request merged**